ruby/vampires.rb

```diff
@@ -0,0 +1,98 @@
+#Driver code
+#Process multiple employees
+puts "How many employees will be processed?"
+ee_num = gets.chomp.to_i
+#initiating loop to process multi employees. It wraps code
+count = 0
+while count < ee_num
+
+# Program to ask new employee:
+#Some known vampires are in the area, and we can check names against the Werewolf Intelligence Bureau database for their aliases.
+
+puts "What is your name?"
+name = gets.chomp.capitalize #entered info is capitalized as names usually are
+puts "Welcome, " + name + "." #print to screen to confirm info entering program
+
+# This is to try to trick the vampire, who is likely several hundreds of years old. If an employee gives an age and a year of birth that don’t line up mathematically, that employee might be a vampire.
+
+puts "How old are you?"
+age = gets.chomp.to_i #enters as integer so can compare with year remainder
+puts "Ok, " + age.to_s + " is a great age." #print to screen to confirm info entering program
+
+puts "What year were you born?"
+year = gets.chomp.to_i #change to integer to subtract from current year
+puts "Well, " + year.to_s + " was a very good year." #print to screen to confirm info entering program
+
+# subtract reported year from current year to compare to reported age
+real_age = 2017 - year
+  if age == real_age
```

Comment:
When we use the comparison operator we get a true or a false value back as a return, therefore isntead of saying set this variable to true if this thing is true we can just say set this variable to this conditional.

`real_age = age == (2017 - year)`